Humans_Engine/Query/CvalueAnalysis.cs

```diff
@@ -114,6 +114,22 @@ private static double GetRowWidth(Spectator current, Spectator nearest, Vector r
         return width;
         }
         /***************************************************/
+        private static double GetRowWidth(Spectator current, Spectator nearest, Point focal, Vector rowV)
```

Comment:
this method appears unnecessary as the previous calculation of row width has been test and works